### PR TITLE
Cleanup before 0.2.0

### PR DESCRIPTION
**Pull request by under-Peter**

### CODE REVIEW

File: test/EinRule.jl [hunk 1]
```diff
@@ -6,80 +6,116 @@ using OMEinsum: match_rule, PairWise, Sum, Tr, DefaultRule,
 @testset "match rule" begin
     ixs = ((1,2), (2,3))
     iy = (1,3)
+    @isdefined(pmobj) && next!(pmobj)
```
Comment (GiggleLiu) [inline test/EinRule.jl]: This kind of irrelevant code is confusing for someone who want to contribute this project. It should only appear in runtests.jl

Comment (GiggleLiu) [reply test/EinRule.jl]: Or is it possible to insert this line automatically by creating a new macro like @progtest

Comment (under-Peter) [reply test/EinRule.jl]: I'll think about it (probably don't want to add a new dependency though) but for the moment I removed it.

Comment (GiggleLiu) [reply test/EinRule.jl]: Good. If this PR is ready, please remember to remove WIP.

**Pull request merged**